interfaces/matlab/toolbox/@Kinetics/getDeltaEnthalpies.m

```diff
@@ -0,0 +1,13 @@
+function dH = getDeltaEnthalpies(a)
+% getDeltaEnthalpies  Get the enthalpy of reaction for each reaction.
+% dH = getDeltaEnthalpies(a)
+%
+% :param a:
+%     Instance of class :mat:func:`Kinetics` (or another
+%     object deriving from Kinetics)
+%     for which the net stoichiometric coefficients are desired.
+% :return:
+%     Returns a vector of the enthalpy of reactions for each reaction.  If a 
+%
+
+dH = kinetics_get(a.id, 17, 0);
```

Comment:
Should be one blank line at the end of the file

Reply:
Ha - opening in atom fixed it for me, automatically (the original was created in VS Code).

Reply:
Yes, its annoying that's not the default in VS Code.